spec/models/message_spec.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: messages
+#
+#  id               :bigint           not null, primary key
+#  messageable_type :string
+#  text             :text             not null
+#  created_at       :datetime         not null
+#  updated_at       :datetime         not null
+#  messageable_id   :bigint
+#  user_id          :bigint
+#
+# Indexes
+#
+#  index_messages_on_messageable  (messageable_type,messageable_id)
+#  index_messages_on_user_id      (user_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (user_id => users.id)
+#
+require 'rails_helper'
+
+RSpec.describe Message, type: :model do
+  let(:user) { FactoryBot.create(:user) }
+  let(:message) { FactoryBot.build(:message, user: user) }
+  let(:empty_message) { FactoryBot.build(:message, :empty, user: user) }
+  let(:message_without_user) { FactoryBot.build(:message, :without_user, user: user) }
```

Comment:
If your FactoryBot setup is correct, you should be able to omit `FactoryBot.` calls and just write `build` / `create`. Value your time :)